src/scaict_uwu/service_container.py

```
@@ -0,0 +1,91 @@
+
+        if not name in self._services_being_created:
+            sys.exit(
+                "RecursiveServiceDependencyException "
+                + "Circular dependency when creating service!"
+            )
+
+        self._services_being_created[name] = True
+
+        service = self._service_instantiators[name](self)
+
+        return service
```

Ruff (0.8.0)

Unnecessary assignment to `service` before `return` statement

Remove unnecessary assignment

(RET504)